src/routes/view-content/[guideId]/[bibleSection]/feedback/UserInfoModal.svelte

```diff
@@ -0,0 +1,106 @@
+<script lang="ts">
+    import { _ as translate } from 'svelte-i18n';
+    import XMarkIcon from '$lib/icons/XMarkIcon.svelte';
+    import { apiUrl } from '$lib/data-cache';
+    import FullPageSpinner from '$lib/components/FullPageSpinner.svelte';
+    import { getAdditionalProperties } from '$lib/logger';
+    import { currentLanguageInfo } from '$lib/stores/language.store';
+    import ContactOptions from './ContactOptions.svelte';
+    import { trapFocus } from '$lib/utils/trap-focus';
+    import { FeedbackType, FormType } from '$lib/types/contact-info';
+
+    export let close: () => void;
+    export let isOpen: boolean;
+
+    let loading = false;
+    let submitDisabled = true;
+    let name = '';
+    let contactType = '';
+    let contactValue = '';
+
+    $: !contactType && (contactValue = '');
+
+    $: name && contactValue && contactType ? (submitDisabled = false) : (submitDisabled = true);
+
+    async function handleSubmit() {
+        loading = true;
+        const metaData = {
+            additionalProperties: getAdditionalProperties(),
+            currentLanguage: {
+                ...$currentLanguageInfo,
+            },
+        };
+
+        const postData = {
+            name,
+            contactType: contactType,
+            contactValue: contactValue,
+            feedback: JSON.stringify(metaData),
+            feedbackType: FeedbackType.PromptFeedbackForm,
+        };
+
+        try {
+            const response = await fetch(apiUrl('feedback/bible-well'), {
```

Comment:
We may want to update this so it can work offline like the other form. I know it wasn't in requirements but seems like since this could pop up whenever it would make sense: https://github.com/BiblioNexusStudio/well-web/blob/master/src/routes/view-content/[guideId]/[bibleSection]/feedback/ResourceFeedbackModal.svelte#L71